packages/manager/modules/nutanix/src/dashboard/general-info/controller.js

```diff
@@ -0,0 +1,6 @@
+export default class NutanixGeneralInfoCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
```

Comment:
Probably, can be removed, since it is not used

Reply:
it will be required while working on general-info